Source/Core/Core/HW/DSPLLE/DSPSymbols.h

```diff
@@ -23,7 +23,6 @@ class DSPSymbolDB : public SymbolDB
 
 extern DSPSymbolDB g_dsp_symbol_db;
```

Comment:
I don't think g_dsp_symbol_db can contain anything after this? Meaning also that a lot of the debugger code is dead...sad :(